### DISCUSSION TITLE
Recover login details by email

### DISCUSSION THREAD

**Discussion by orschiro** [joomla/joomla-cms#44603] [General]
Hi, 

Why is this not possible?

This is pretty much standard these days. 

I don't know any other system where this is not possible.

<img width="346" alt="image" src="https://github.com/user-attachments/assets/b05c0c6d-0d6b-435a-a5d0-923eb9ef4ad3">

https://docs.joomla.org/How_do_you_recover_or_reset_your_admin_password%3F

**Comment by Hackwar** [comment]
Because if you are an admin, you have other possibilities to do this, for example going into the database directly. At the same time, it is a security issue if you can simply reset the password as super admin when someone got access to your mail account.

**Comment by orschiro** [comment]
Thanks, Hannes!

But what about non-super admin users?

**Comment by Hackwar** [comment]
Those can do that over the frontend login screen.

**Comment by orschiro** [comment]
Can you pass me the link to the frontend login screen?

Thank you!

**Comment by Fedik** [comment]
Check for the login form on the site, there should be a link to recovery.
If it does not exist, then please contact to administrator of your site (user who is super admin on your site).

**Comment by orschiro** [comment]
I only know one login form: website.com/administrator

This is what we use as admins. 

Is this the same for non-admins?

**Comment by Fedik** [comment]
Yes, /administrator can be used by people who allowed to login in Admin section.
You better contact to the administrator of your site

**Comment by orschiro** [comment]
That's my point.

If a normal author (non admin) can sign in via /administrator but forgot
his password, there should be an option for him to restore his password via
email.

**Comment by Fedik** [comment]
I agree
